src/Round.js

```diff
@@ -1,28 +1,42 @@
+import data from './data';
```

Comment:
Once you are using `fetch`, you should not need to include this `data` line here because the data should ultimately come from the `fetch` call.